recipes/perl-btlib/0.19/build.sh

```diff
@@ -11,7 +11,6 @@ elif [ -f Makefile.PL ]; then
     # Make sure this goes in site
     perl Makefile.PL INSTALLDIRS=site
     make
-    make test
```

Comment:
Why is this needed? I think it is actually good to have it, it increase the testing of the package isn't it?